test/router.test.ts

```diff
@@ -1,10 +1,9 @@
 import Layer from '@koa/router/lib/layer';
-import InMemoryArticles from '../src/adaptors/in-memory-articles';
 import createRouter from '../src/router';
 import Routes from '../src/routes';
 
 describe('router', (): void => {
-  const router = createRouter(new InMemoryArticles());
+  const router = createRouter();
```

Comment:
sounds good that to create a router you don't need an `Articles` implementation :thinking: